rawpheno.module

```diff
@@ -326,6 +326,10 @@ function rawpheno_preprocess_rawpheno_upload_form_master(&$variables, $hook) {
   $variables['rel_url'] = url('phenotypes/raw/instructions');
   // Page Title
   $variables['page_title'] = variable_get('rawpheno_upload_title');
+  // Support Email.
+  $support_email = '';
+  drupal_alter('rawpheno_support_email', $support_email);

```

Comment:
There should be an API function for retrieving the support email. Furthermore, the code in kp_nodes determining the default support email (https://github.com/UofS-Pulse-Binfo/kp_nodes/pull/7/files#diff-ea83b3e1886aaaeac403bd6114856bd6R553)  should be in this rawphenotypes API function --otherwise only we are given a default ;-)